examples/docker/modin-omnisci/Dockerfile

```diff
@@ -11,17 +11,24 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-FROM ubuntu:18.04
-ENV http_proxy ${http_proxy}
-ENV https_proxy ${https_proxy}
-ENV no_proxy ${no_proxy}
-ENV MODIN_BACKEND "omnisci"
-ENV MODIN_EXPERIMENTAL "true"
+# Build image from this dockerfile like this:
+# docker build -t modin-omnisci:latest .
+
+FROM ubuntu:20.04
+
+# Proxy settings
+ENV http_proxy=${http_proxy}
+ENV https_proxy=${https_proxy}
+ENV no_proxy=${no_proxy}
```

Comment:
Will they be defined without appropriate use `--build-arg`?

Reply:
Yes, it works for me.